### PR TITLE
setup.xml: remove the extra and add the missing chars

### PR DESCRIPTION
**Pull request by mmalferov**

### CODE REVIEW

File: reference/gmagick/setup.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
    some features and formats are not supported for these versions.
    Windows version of GraphicsMagick is available from the GraphicsMagick
    website. To read about the difference between ImageMagick and GraphicsMagick,
-   check this out: <link xlink:href="&url.gmagick.imagemagickdiff.faq;">FAQ</link> .
+   check this out: <link xlink:href="&url.gmagick.imagemagickdiff.faq;">FAQ</link>.
```
Comment (Girgias) [inline reference/gmagick/setup.xml:15]:
```suggestion
   check the <link xlink:href="&url.gmagick.imagemagickdiff.faq;">FAQ</link>.
```

File: reference/gmagick/setup.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
   </note>
   <para>
     For Windows, the build of the Gmagick extension is available
-    <link xlink:href="&url.gmagick.winbuilds;">here</link>
+    <link xlink:href="&url.gmagick.winbuilds;">here</link>.
```
Comment (Girgias) [inline reference/gmagick/setup.xml:31]: Same comment as another PR, changing the "here" wording might be a good idea

Comment (Girgias) [reply reference/gmagick/setup.xml:31]: @cmb69 you said the above link doesn't work anymore?

Comment (cmb69) [reply reference/gmagick/setup.xml:31]: The URL is http://valokuva.org/magick/. Chrome says: DNS_PROBE_FINISHED_NXDOMAIN. Does it work for you?

Comment (Girgias) [reply reference/gmagick/setup.xml:31]: Nope this is also dead for me

**Comment by Girgias** [comment]
Can always fix the link in another PR.

**Pull request merged**